AdapterFacade/Information.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.NetMentoring.AdapterFacade
+{
+    class Information<T> : IElements<T>
+    {
+        private readonly List<T> _list = new List<T>();
+
+        public Information(List<T> list)
+        {
+            _list = list ?? throw new ArgumentNullException("Cannot create Information class");
```

Comment:
Exception message text does not correspond to what is actually happening:
- ArgumentNullException(nameof(list));

Reply:
ok